cli/morphir.js

```diff
@@ -4,17 +4,22 @@ const util = require('util')
 const fs = require('fs')
 const readdir = util.promisify(fs.readdir)
 const readFile = util.promisify(fs.readFile)
-const worker = require('./Morphir.Elm.CLI').Elm.Morphir.Elm.CLI.init()
+const packageDefWorker = require('./Morphir.Elm.CLI').Elm.Morphir.Elm.CLI.init()
+const elmEncoderWorker = require('./Morphir.Elm.EncodersCLI').Elm.Morphir.Elm.EncodersCLI.init()
```

Comment:
Longer term I think we should just have one worker and use it as a facade to consolidate all the functionality but I'm guessing this was meant to be a temporary solution anyway so OK for now.

Reply:
That is a good suggestion. I did think of that but like you said, for the purposes of getting something out quicker I fell short of consolidating it.